src/styles/list.scss

```diff
@@ -0,0 +1,14 @@
+.title-container {
+    display: flex;
+    justify-content: space-between
+}
+
+.nutrients-container {
```

Comment:
Name should be more generic if it is shared `list.scss` file or rename file with `nutrition-list.scss`

Reply:
Done